renode-linux-runner-docker/payload/config

```diff
@@ -2,4 +2,13 @@
 
 # Configure linux environment before running python script
 
-exec /run-in-renode.py "$@"
\ No newline at end of file
+TAP_INTERFACE=tap0
+HOST_INTERFACE=eth0
+
+ip tuntap add mode tap ${TAP_INTERFACE} && \
+ip addr add "172.16.0.1/16" dev ${TAP_INTERFACE} && \
+ip link set up dev ${TAP_INTERFACE} && \
+iptables --table nat --append POSTROUTING --out-interface ${HOST_INTERFACE} -j MASQUERADE && \
+iptables --append FORWARD --in-interface ${TAP_INTERFACE} -j ACCEPT
```

Comment:
I think this could use a comment that this is all for an Internet connection in Renode. Obviously the commit message says that, but when you read it you don't necessarily look at blame. Also people reading this might not be great at networking, so more comments wouldn't hurt.